sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -111,6 +111,7 @@
           <zeebe:header key="studentDataAdditionalTransportPlacement" value="$.data.additionalTransportPlacement" />
           <zeebe:header key="applicationStatus" value="$.applicationStatus" />
           <zeebe:header key="applicationHasNOAApproval" value="$.hasNOAApproval" />
+          <zeebe:header key="appealsStudentDependantsData" value="$.appeals.studentDependantsAppeal.submittedData.dependants[*][&#39;dateOfBirth&#39;,&#39;declaredOnTaxes&#39;,&#39;attendingPostSecondarySchool&#39;]" />
```

Comment:
Please have the appeals key added next to other appeal keys. 
![image](https://github.com/user-attachments/assets/2384be2b-6ecc-4242-8d00-4f1b5de4f73d)